python/cudf/cudf/core/column/column.py

```diff
@@ -2299,7 +2299,62 @@ def full(size: int, fill_value: ScalarLike, dtype: Dtype = None) -> ColumnBase:
     return ColumnBase.from_scalar(cudf.Scalar(fill_value, dtype), size)
 
 
-def _concat_columns(objs: "MutableSequence[ColumnBase]") -> ColumnBase:
+def _copy_type_metadata_from_arrow(
```

Comment:
Bad merge? I thought this was removed.

Reply:
Indeed, good catch! Fixed, and also changed title